main.py

```diff
@@ -512,20 +404,15 @@ async def give(ctx:SlashContext, user:discord.User, amount:int):
 @commands.cooldown(1, 60, commands.BucketType.user)
 async def rob(ctx:SlashContext, user:discord.User):
     if plugins.economy == False: pass
-    chance:int = random.randint(1, 100)
-    if (currency['wallet'][str(user.id)] < 5000):
-        await ctx.reply('They has less than 5000 coins on them. Don\'t waste your time...')
-        return
-    elif (currency['wallet'][str(ctx.author.id)] < 5000):
-        await ctx.reply('You have less than 5k coins in your wallet. Play fair dude.')
-        return
-    if (chance <= 50):
-        x:int = random.randint(5000, currency['wallet'][str(user.id)])
+    if currency['wallet'][str(user.id)] < 5000: return await ctx.reply('They has less than 5000 coins on them. Don\'t waste your time...') 
+    elif currency['wallet'][str(ctx.author.id)] < 5000: return await ctx.reply('You have less than 5k coins in your wallet. Play fair dude.') #what..? wheres logic?? i mean irl you can rob someone without havin any money at all
```

Comment:
Technically, robbing someone without having any money is possible, but there also has to be a failure penalty. If it was still 5000 coins from wallet, then the user would be in "debt"

Reply:
if wallet == 0: return
else: random.randint(round(wallet/10, 0), wallet)

Reply:
I'll try it with this too. Lets see what works